Makefile

```diff
@@ -28,7 +28,7 @@ ifndef DOCKER_HUB_REGISTRY_IMG
     $(warning DOCKER_HUB_REGISTRY_IMG not defined, using '$(DOCKER_HUB_REGISTRY_IMG)' instead)
 endif
 ifndef BASE_REGISTRY_IMG
-    BASE_REGISTRY_IMG := docker.io/portworx/px-operator-registry:23.10.3
+    BASE_REGISTRY_IMG := docker.io/portworx/px-operator-registry:25.1.0
```

Comment:
This should be the previous release version.

Reply:
fixed